src/caustics/cosmology.py

```diff
@@ -377,6 +386,8 @@ def critical_density(
         -------
         torch.Tensor
             Critical density at redshift z.
+            *Unit: Msol/Mpc^3*
```

Comment:
Is this supposed to be `solMass`?

Reply:
It was mentioned MSol in someplaces and solMass in some places. Updated everything to solMass